tracer/operation/operation_test.go

```diff
@@ -273,6 +274,26 @@ func (s *MockStateDB) GetSubstatePostAlloc() txcontext.WorldState {
 	return nil
 }
 
+func (s *MockStateDB) CreateContract(addr common.Address) {
+	panic("CreateContract not supported in mock")
+}
+
+func (s *MockStateDB) Selfdestruct6780(addr common.Address) {
+	panic("Selfdestruct6780 not supported in mock")
+}
+
+func (s *MockStateDB) GetStorageRoot(addr common.Address) common.Hash {
+	panic("GetStorageRoot not supported in mock")
+}
+
+func (s *MockStateDB) SetTransientState(addr common.Address, key common.Hash, value common.Hash) {
+	panic("SetTransientState not supported in mock")
+}
+
+func (s *MockStateDB) GetTransientState(addr common.Address, key common.Hash) common.Hash {
+	panic("GetTransientState not supported in mock")
+}
```

Comment:
You can use [this](https://github.com/Fantom-foundation/Aida/pull/1103) PR as inspiration

Reply:
I plan to add new operations for tracer and profiler in a separate PR. Some of them has already been done by your PR.